xml/System.Xml/XmlDocumentXPathExtensions.xml

```diff
@@ -41,9 +41,9 @@
         <Parameter Name="document" Type="System.Xml.XmlDocument" RefType="this" />
       </Parameters>
       <Docs>
-        <param name="document">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="document">Document to create XPath navigator from.</param>
+        <summary>Creates a new <see cref="T:System.Xml.XPath.XPathNavigator" /> object for navigating this document.</summary>
+        <returns>An <see cref="T:System.Xml.XPath.XPathNavigator" /> object.</returns>
```

Comment:
Maybe we can keep it consistent and use natural language, which is usually preferred, instead of crefs to the types (the link to the type will already be showing up in MS Docs):
```suggestion
        <summary>Creates a new XPath navigator object for navigating the specified document.</summary>
        <returns>An XPath navigator object.</returns>
```

Reply:
those are copied from original type. This is a dead class at this point so I wouldn't spend too much time on it